src/Microbiome.jl

```diff
@@ -32,18 +40,23 @@ export
     taxfilter,
     taxfilter!
 
+using Reexport
+@reexport using SpatialEcology
+@reexport using Distances
+
 using RecipesBase
 using StatPlots
 using StatsBase
-using Distances
 using Colors
 using DataFrames
 
+import SpatialEcology.@forward_func
+import SpatialEcology.summary
```

Comment:
No idea. I wasn't sure how to modify it since the way it currently prints doesn't make sense with my feature names, but I didn't want to override your version. Once we have ecobase, I'll define a custom microbiome type I think, but for now it doesn't seem important.